src/test/ui/fmt/format-with-yield-point.rs

```diff
@@ -0,0 +1,23 @@
+// check-pass
+// edition:2021
+
+macro_rules! m {
+    () => {
+        async {}.await
+    };
+}
+
+async fn with_await() {
+    println!("{} {:?}", "", async {}.await);
+}
+
+async fn with_macro_call() {
+    println!("{} {:?}", "", m!());
+}
```

Comment:
Let's test with a macro call statement as well:

```suggestion
async fn with_macro_call() {
    println!("{} {:?}", "", m!());
}

async fn with_macro_call_stmt() {
    println!("{} {:?}", "", { m!(); });
}
```

or something along those lines

Reply:
Added, and also `{ m!{} }`.